.script/dep.sh

```diff
@@ -8,4 +8,8 @@ go get -u github.com/colinmarc/hdfs
 go get -u golang.org/x/net/context
 go get -u google.golang.org/grpc
 go get -u github.com/golang/protobuf/proto
-go get -u github.com/MSOpenTech/azure-sdk-for-go/storage
+go get -u golang.org/x/tools/cmd/vet
```

Comment:
These script is used to track high level deps.
Those sub-deps will be downloaded automatically.